benchmarks/litdata/stream_raw_imagenet.py

```diff
@@ -0,0 +1,144 @@
+"""Benchmark streaming raw dataset with litdata."""
+
+import argparse
+import os
+import shutil
+import time
+from contextlib import suppress
+
+import torch
+import torchvision.transforms.v2 as T
+from torch.utils.data import DataLoader
+from torchvision.io import ImageReadMode, decode_jpeg
+from tqdm import tqdm
+
+from litdata.streaming.raw_dataset import StreamingRawDataset
+
+
+def clear_cache_dir(cache_dir: str) -> None:
+    """Clear the cache directory."""
+    with suppress(Exception):
+        if os.path.isdir(cache_dir):
+            shutil.rmtree(cache_dir)
+
+
+def deserialize_jpeg(data: bytes) -> torch.Tensor:
+    """Deserialize JPEG bytes to a tensor."""
+    arr = torch.frombuffer(data, dtype=torch.uint8)
+    with suppress(RuntimeError):
+        return decode_jpeg(arr, mode=ImageReadMode.RGB)  # RGB
+
+    import io
+
+    from PIL import Image
+
+    img = Image.open(io.BytesIO(data)).convert("RGB")
```

Comment:
Torchvision supports image_read and does this logic under the hood I believe.

Reply:
added `decode_image`.